protocols/util/factories.py

```diff
@@ -0,0 +1,47 @@
+import factory.fuzzy
+import itertools
+from protocols.ga4gh_3_0_0 import Variant, Call
+from .avro_factory import FactoryAvro
+
+
+class GA4GHVariantFactory(FactoryAvro):
+    def __init__(self, *args, **kwargs):
+        super(GA4GHVariantFactory, self).__init__(*args, **kwargs)
+
+    class Meta:
+        model = Variant
+    start = factory.fuzzy.FuzzyInteger(1, 10000000)
```

Comment:
I will prepare some test, trying to do complex things, in the meantime https://factoryboy.readthedocs.io/en/latest/index.html
 and @gerlv has experience with the library (you can check panelApp tests)